### ISSUE TITLE
Vanadium Browser crashes AppManager

### ISSUE THREAD

**Issue by S7venLights** [MuntashirAkon/AppManager#970]
On Graphene OS using Vanadium browser which is a System app by default. 

Crash Report: 
java.lang.RuntimeException: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean io.github.muntashirakon.AppManager.apk.ApkFile.isSplit()' on a null object reference
    at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel.lambda$loadTagCloud$0(AppInfoViewModel.java:216)
    at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel$$ExternalSyntheticLambda0.run(Unknown Source:2)
    at android.os.Handler.handleCallback(Handler.java:938)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:201)
    at android.os.Looper.loop(Looper.java:288)
    at android.app.ActivityThread.main(ActivityThread.java:7876)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
    at com.android.internal.os.ExecInit.main(ExecInit.java:49)
    at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
    at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
 Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean io.github.muntashirakon.AppManager.apk.ApkFile.isSplit()' on a null object reference
   at io.github.muntashirakon.AppManager.details.AppDetailsViewModel.getSplitCount(AppDetailsViewModel.java:978)
   at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel.loadTagCloud(AppInfoViewModel.java:135)
   at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.loadPackageInfo(AppInfoFragment.java:1531)
   at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.$r8$lambda$oxlKbqiAsxIkEG69gntC-jrgnXk(Unknown Source:0)
   at io.github.muntashirakon.AppManager.details.info.AppInfoFragment$$ExternalSyntheticLambda104.run(Unknown Source:2)
   at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
   at java.lang.Thread.run(Thread.java:920)

Device Info:
App version: 3.0.4
App version code: 416
Android build version: 2022122900
Android release version: 12
Android SDK version: 32
Android build ID: SP1A.210812.016.C2.2022122900
Device brand: google
Device manufacturer: Google
Device name: blueline
Device model: Pixel 3
Device product name: blueline
Device hardware name: blueline
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-US
In-App Language: auto
Mode: auto
Inferred Mode: no_root

### Android Crash Report

```
type: crash
osVersion: google/blueline/blueline:12/SP1A.210812.016.C2/2022122900:user/release-keys
package: io.github.muntashirakon.AppManager:416
process: io.github.muntashirakon.AppManager
processUptime: 13121 ms

java.lang.RuntimeException: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean io.github.muntashirakon.AppManager.apk.ApkFile.isSplit()' on a null object reference
	at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel.lambda$loadTagCloud$0(AppInfoViewModel.java:216)
	at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel$$ExternalSyntheticLambda0.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7876)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean io.github.muntashirakon.AppManager.apk.ApkFile.isSplit()' on a null object reference
	at io.github.muntashirakon.AppManager.details.AppDetailsViewModel.getSplitCount(AppDetailsViewModel.java:978)
	at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel.loadTagCloud(AppInfoViewModel.java:135)
	at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.loadPackageInfo(AppInfoFragment.java:1531)
	at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.$r8$lambda$oxlKbqiAsxIkEG69gntC-jrgnXk(Unknown Source:0)
	at io.github.muntashirakon.AppManager.details.info.AppInfoFragment$$ExternalSyntheticLambda104.run(Unknown Source:2)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:920)

```

**Comment by MuntashirAkon** [comment]
Seems to be a duplicate of #951

**Comment by MuntashirAkon** [comment]
Can you upload the APK file here?

**Comment by S7venLights** [comment]
Github doesn't allow APK attatchments so here (link expires in 2 weeks): https://send.internxt.com/download/7e5b036c-cdbf-47a0-8ae0-4f0f2baeacc7?code=547f9b98a2826f3b4027ba2384e541ec66f21a27ec7e06500d0c4567f953098d

PS vadadium launched in the app manager version two releases ago.

**Comment by MuntashirAkon** [comment]
You need to send the APK files exported by App Manager as it has certain library dependencies (i.e. trichrome) that aren't available in my device.

**Comment by S7venLights** [comment]
I did use app manager to export that APK. 
Don't know what to say? 

Maybe it's because Vanadium is a system app in Graphene OS??

**Comment by MuntashirAkon** [comment]
> I did use app manager to export that APK.

Ah, I see that you've used v3.0.4. In v3.1.0-alpha01, a new feature is added that allows exporting an APK along with all the required dependencies provided by the system. You'll have to wait for a few months before it will be available in the stable version.

**Comment by MuntashirAkon** [comment]
Fixed in 9810cb4ad7965469ad78ad15de76caa7c4f2eb94
